xero-finance.yaml

```diff
@@ -2458,13 +2364,12 @@ components:
           x-is-money: true
         indicativeEndBalance:
           type: number
-          description: |
```

Comment:
what is this | char? Did it had any significance

Reply:
yq automatically formats by adding \n at teh end replacing |